src/main/java/training/TrainingRobotManager.java

```diff
@@ -1,14 +1,27 @@
 package training;
 
+import edu.wpi.first.math.geometry.Rotation2d;
+import training.commands.elbow.MoveElbowToAngle;
+import training.subsystems.elbow.RobotTypes;
 import utils.DefaultRobotManager;
+import utils.joysticks.JoystickPorts;
+import utils.joysticks.SmartJoystick;
 
 public class TrainingRobotManager extends DefaultRobotManager {
 
 	private Robot robot;
 
+	private SmartJoystick smartJoystick;
```

Comment:
you don't really need to save this as a variable in the class, because you only use it in one function